jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+---
+title: Python Developers (x2 roles)
```

Comment:
It might be best to omit the (x2 roles) bit, since each candidate will only apply for a single role. You can just leave the ad online until both positions are filled.